src/NzbDrone.Core/Indexers/Definitions/ImmortalSeed.cs

```diff
@@ -171,12 +171,20 @@ private IEnumerable<IndexerRequest> GetPagedRequests(string term, int[] categori
         {
             var searchUrl = Settings.BaseUrl + "browse.php";
 
-            //TODO - Actually map some categories here
             if (term.IsNotNullOrWhiteSpace())
             {
                 searchUrl += string.Format("?do=search&keywords={0}&search_type=t_name&category=0&include_dead_torrents=no", WebUtility.UrlEncode(term));
```

Comment:
We should remove category=0 here if it's not needed now

Reply:
Weirdly, "&category=0" is still needed, but it's not actually categories.

Reply:
Ah... Odd